deployments/container/Dockerfile.ubi8

```diff
@@ -39,16 +39,15 @@ ENV PATH $GOPATH/bin:/usr/local/go/bin:$PATH
 
 WORKDIR /build
 COPY . .
-RUN GOOS=linux go build -o /artifacts/nvidia-vgpu-dm ./cmd/nvidia-vgpu-dm
-RUN GOOS=linux go build -o /artifacts/k8s-nvidia-vgpu-dm ./deployments/container/
+RUN make PREFIX=/artifacts cmds
 
 
 FROM nvcr.io/nvidia/cuda:${CUDA_VERSION}-base-${BASE_DIST}
 
 ENV NVIDIA_VISIBLE_DEVICES=void
 
 COPY --from=build /artifacts/nvidia-vgpu-dm /usr/bin/nvidia-vgpu-dm
-COPY --from=build /artifacts/k8s-nvidia-vgpu-dm /usr/bin/k8s-nvidia-vgpu-dm
+COPY --from=build /artifacts/nvidia-k8s-vgpu-dm /usr/bin/nvidia-k8s-vgpu-dm
```

Comment:
Just to confirm that no other components depend on this name specifically?

Reply:
Correct, no others depend on this name.